pkg/api/v1/types.go

```diff
@@ -354,6 +371,11 @@ const (
 	// used for PersistentVolumes where the bound PersistentVolumeClaim was deleted
 	// released volumes must be recycled before becoming available again
 	VolumeReleased PersistentVolumePhase = "Released"
+	// used for PersistentVolumes that have been recycled and can be made available
+	// again to new PersistentVolumeClaims
+	VolumeRecycled PersistentVolumePhase = "Recycled"
```

Comment:
We'll need to redesign the entire PV feature, then, because as you say, I'm driving behavior on phase in the binder.

What is the best past forward in the short term?  Is this PR acceptable as-is (e2e test pending, of course)?  It is late in the game to change anything.